doc/devel/interface_specs.rst

```diff
@@ -7,19 +7,18 @@ Interface Specifications
 Before you start
 ----------------
 
-Nipype is a young project maintained by an enthusiastic group of developers.
-Even though the documentation might be sparse or cryptic at times we strongly
-encourage you to contact us on the official nipype developers mailing list in
-case of any troubles: nipy-devel@neuroimaging.scipy.org (we are sharing a
-mailing list with the nipy community therefore please add ``[nipype]`` to the
-messsage title).
+Nipype is maintained by an enthusiastic group of developers, and we're excited to have you join!
+In case of trouble, we encourage you to post on `NeuroStars <https://neurostars.org>`_ with the `nipype` tag.
+NeuroStars.org is a platform similar to StackOverflow but dedicated to neuroinformatics.
+You can also post on the nipype developers mailing list: http://mail.python.org/mailman/listinfo/neuroimaging.
+We are sharing a mailing list with the nipy community therefore please add ``[nipype]`` to the message title.
 
 
 Overview
 --------
 
-We're using the `Enthought Traits
-<http://code.enthought.com/projects/traits/>`_ package for all of our
+We're using the `Traits, formerly known as Enthought Traits
```

Comment:
I'd parenthesize the "formerly known as Enthought Traits" part, and not include it in the link text.